### ISSUE TITLE
New version released  - Release v2.2.1, but CHANGELOG not updated

### ISSUE THREAD

**Issue by eazygood** [jasonsims/aws-cloudfront-sign#69]
Hey,
Noticed that CHANGELOG not updated  and also npmjs does not  know anything about new version.
Please check https://www.npmjs.com/package/aws-cloudfront-sign

Thank you

**Comment by deathstar1708** [comment]
Hi @jasonsims , could you help us release 2.2.1 to npm so that the latest changes done especially on the lodash vulnerability is released to use. Thank You.

**Comment by seanmurph** [comment]
+1 however @deathstar1708 @eazygood in the meantime you can do:

```
npm i https://github.com/jasonsims/aws-cloudfront-sign.git#v2.2.1
```

Also worth noting AWS SDK v3 implements this functionality natively. My project is on v2 where we'd otherwise have to reinvent this.

**Comment by jasonsims** [comment]
2.2.1 has been released along with 3.0.0
